src/modules/app/routeConfigs/events.js

```diff
@@ -0,0 +1,51 @@
+// @flow
+
+import RouteConfig from './RouteConfig'
+import type { Route } from 'redux-first-router'
+import type { AllPayloadsType, GetLanguageChangePathParamsType, GetPageTitleParamsType } from './RouteConfig'
+import Payload from '../../endpoint/Payload'
+import EventModel from '../../endpoint/models/EventModel'
+
+type EventsRouteParamsType = {|city: string, language: string|}
+type RequiredPayloadsType = {|events: Payload<Array<EventModel>>|}
+
+export const EVENTS_ROUTE = 'EVENTS'
```

Comment:
These const exports should be part of RouteConfig imo

Reply:
@Schedulaar what do you think? If think its okay to export the route name separately.

Reply:
I think they should only be part of the RouteConfigs...